content/grants/grants-faqs.mdx

```diff
@@ -0,0 +1,25 @@
+<FAQItem title="What are the requirements to apply for a grant?">
+To be eligible for a grant, you must provide KYC information. All grants are subject to the laws of the recipient's governing country.
```

Comment:
I guess we'll have to find a way to react-intl these, but we're not doing it consistently yet anyway